plenum/test/consensus/conftest.py

```diff
@@ -100,3 +106,53 @@ def _view_change(view_no: int):
         return vc
 
     return _view_change
+
+
+@pytest.fixture(params=[True, False])
+def is_master(request):
+    return request.param
+
+
+@pytest.fixture()
+def internal_bus():
+    def rp_handler(ib, msg):
+        ib.msgs.setdefault(type(msg), []).append(msg)
+
+    ib = InternalBus()
+    ib.msgs = {}
+    ib.subscribe(RequestPropagates, rp_handler)
+    return ib
+
+
+@pytest.fixture()
+def external_bus():
+    send_handler = Mock()
+    return ExternalBus(send_handler=send_handler)
```

Comment:
Actually there is already `MockNetwork` helper class which can be used instead of this fixture. As a bonus it already contains `sent_messages` field, so it is straightforward to check whether some service sent some messages.